Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/clip.h

```diff
@@ -257,11 +257,11 @@ clip_to_bbox(const Plane_3& plane,
 
   // take the convex hull of the points on the negative side+intersection points
   // overkill...
-  Polyhedron_3<Geom_traits> P;
-  CGAL::convex_hull_3(points.begin(), points.end(), P);
-  copy_face_graph(P, tm_out,
+  TriangleMesh ch_tm;
+  CGAL::convex_hull_3(points.begin(), points.end(), ch_tm);
+  copy_face_graph(ch_tm, tm_out,
                   Emptyset_iterator(), Emptyset_iterator(), Emptyset_iterator(),
-                  get(vertex_point, P), vpm_out);
+                  get(vertex_point, ch_tm), vpm_out);
```

Comment:
`Polyhedron` is not in the set of dependencies of PMP, even before this commit. That is strange.

(Maybe because `<CGAL/Polygon_mesh_processing/clip.h>` is not in the set of documented headers.)

Reply:
exactly, it was not documented. I think this also holds for Convex_hull_3